### ISSUE TITLE
TypeError: Cannot read property 'dependents' of undefined

### ISSUE THREAD

**Issue by wing-king** [changesets/changesets#1133]

**Comment by wing-king** [comment]
<img width="475" alt="image" src="https://user-images.githubusercontent.com/18393778/230965681-a084f10f-34e0-4166-9bce-7eb10511be93.png">
pkg.packageJson.name = undefine  is why?

**Comment by ntnyq** [comment]
Encountering this issue too.

[A minimal repro repo](https://github.com/qxy-fe/changesets-issue-repro)

Run `pnpm exec changeset -h` will see the error.

**Comment by HalvorHaugan** [comment]
This seems to have been fixed now.

**Comment by ntnyq** [comment]
Yes, I tried with @changesets/cli v2.27.11, the issue disappears.

This can be closed.
